_summaries/sutskever2014sequence_1.md

```diff
@@ -1,19 +1,48 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: stskever2014sequence
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: saikm200022
+score: 9
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+# **Summary - Sequence to Sequence Learning with Neural Networks, Sutskever, Vinyals, Le; 2014**
+
+## What is the core idea?
+
+Deep Neural Networks are being used today for all kinds of difficult tasks such as image detection and speech recognition. However, DNNs are not suitable for sequence to sequence tasks because the dimensionality of input and output sequences can be arbitrary. This paper suggests a Deep Learning approach that is able to apply DNNs for a sequence to sequence task by bypassing its limitation of requiring fixed dimensionality input and output.
+
+![Alt Text](sutskever2014sequence_1b.PNG)
+
+
+## How is it realized (technically)?
+
+The model was used on the WMT'14 English to French Machine Translation task. The paper trained the model on a subset of 12 million sentences with vocabulary size of 348 million and 304 million words for French and English respectively. 
```

Comment:
The experiment and evaluation protocol are usually considered separate issues to the method designs. Bringing up the dataset details upfront would confuse readers unless there are key parts influencing the method design.

Reply:
That makes sense, I have moved the dataset details to before the experimental results are mentioned.